build.gradle.kts

```diff
@@ -63,39 +47,118 @@ ext {
 
 version = String.format("%s-%s", rootProject.version, buildNumber)
 
-tasks.named<ShadowJar>("shadowJar") {
-    archiveClassifier.set(null as String?)
-    dependencies {
-        relocate("com.cryptomorin.xseries", "net.arcaniax.buildersutilities.xseries") {
-            include(dependency("com.github.cryptomorin:XSeries:9.8.1"))
-        }
-        relocate("org.bstats", "net.arcaniax.buildersutilities.metrics") {
-            include(dependency("org.bstats:bstats-base:3.0.2"))
-            include(dependency("org.bstats:bstats-bukkit:3.0.2"))
-        }
-        relocate("io.papermc.lib", "net.arcaniax.buildersutilities.paperlib") {
-            include(dependency("io.papermc:paperlib:1.0.8"))
-        }
-        relocate("org.incendo.serverlib", "net.arcaniax.buildersutilities.serverlib") {
-            include(dependency("dev.notmyfault.serverlib:ServerLib:2.3.4"))
-        }
-    }
-    minimize()
-}
-
 spotless {
     java {
         licenseHeaderFile(rootProject.file("HEADER.txt"))
         target("**/*.java")
     }
 }
 
-tasks.named<Copy>("processResources") {
-    filesMatching("plugin.yml") {
-        expand("version" to project.version)
+java {
+    toolchain.languageVersion.set(JavaLanguageVersion.of(8))
+    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_1_8
+}
+
+tasks {
+    compileJava {
+        options.release.set(8)
```

Comment:
The code restored is incorrect to the code removed. The toolchain should point to Java 17 as development goal but compile against Java 8 as release goal.

Reply:
Okay not known. I will correct that!

Reply:
> The code restored is incorrect to the code removed. The toolchain should point to Java 17 as development goal but compile against Java 8 as release goal.

The code removed hasn't been restored correctly, TARGET_JVM_VERSION_ATTRIBUTE is still missing, causing a failure in GH actions.